site/content/en/docs/tasks/run_plain_pods.md

```diff
@@ -71,7 +84,11 @@ The resource needs of the workload can be configured in the `spec.containers`.
           cpu: 3
 ```
 
-### c. Limitations
+### c. The "managed" label
```

Comment:
Maybe we should have a section `## Which Pods are managed by Kueue?`, before `Pod definition`

And we would add the paragraph: `Pods that belong to other API resources managed by Kueue are excluded from being queued by `pod` integration.`

`Limitations` should also be `##`, as it's not part of the Pod definition.

But.....  follow up.

/lgtm
/approve